packages/create-app/templates/default-app/packages/backend/Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM node:12-buster
+# This dockerfile builds an image for the backend package.
+# It should be executed with the root of the repo as docker context.
+#
+# Before building this image, be sure to have run the following commands in the repo root:
+#
+# yarn install
+# yarn tsc
+# yarn build
```

Comment:
Same as above, what yarn command actually is this related to is a question a casual code reader may ask